app/helpers/request_info_helper.rb

```diff
@@ -6,13 +6,15 @@ def provision_tab_configuration(workflow)
     return prov_tab_labels, workflow.get_dialog_order
   end
 
-  def prov_grid_data(edit, vms, vm)
+  private
```

Comment:
Not for this PR, but I think _all_ of the method in this helper should be private (I could be wrong, hence not for this PR)